evalml/pipelines/components/transformers/preprocessing/featuretools.py

```diff
@@ -52,12 +61,13 @@ def fit(self, X, y=None):
         Returns:
             self
         """
-        X_ww = infer_feature_types(X)
-        X_ww = X_ww.ww.rename({col: str(col) for col in X_ww.columns})
-        es = self._make_entity_set(X_ww)
-        self.features = dfs(
-            entityset=es, target_dataframe_name="X", features_only=True, max_depth=1
-        )
+        if not self.features:
```

Comment:
nit, but why not use `if not self._passed_in_features`?

Reply:
can change! its because I wrote that logic before I added `self._passed_infeatures` 😄